src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier randomColor = new ColorSupplier();
+        Ball randomlyBall = new Ball();
+        Random random = new Random();
+        for (int i = 0; i < random.nextInt(100); i++) {
+            randomlyBall.setColor(randomColor.getRandomColor());
+            randomlyBall.setNumber(randomColor.getRandomColor().length());
+        }
```

Comment:
I don't get the logic of this loop
can we just calculate a random number and set it once to the ball?